coalib/bearlib/abstractions/Linter.py

```diff
@@ -447,10 +447,13 @@ def run(self, filename, file, **kwargs):
                     FunctionMetadata.filter_parameters(
                         self._get_create_arguments_metadata(), kwargs))
 
-                arguments = (self.get_executable(),) + tuple(
-                    self.create_arguments(
-                        filename, file, config_file,
-                        **create_arguments_kwargs))
+                args = self.create_arguments(filename, file, config_file,
+                                             **create_arguments_kwargs)
+
+                if args is None:  # pragma: no cover
+                    self.warn("The tuple of arguments cannot be None.")
```

Comment:
yep this should be an error.
and yes please return otherwise it still crashes.

And I think you should change the check to a try-error construct:
```
try:
    arguments = tuple(args)
except TypeError:
    self.err("Arguments are not iterable") # <-- make this a bit nicer
    # + you could also fire a debug message here too with additional info on the linter,
    # and what was exactly returned etc.
    return
```